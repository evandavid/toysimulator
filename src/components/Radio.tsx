import * as React from 'react';
import {RadioOuter, RadioInner} from './styled';

const Radio = ({selected}: {selected: boolean}) => {
  return <RadioOuter>{selected && <RadioInner />}</RadioOuter>;
};

export default Radio;
