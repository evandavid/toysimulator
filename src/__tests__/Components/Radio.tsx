import React from 'react';
import renderer from 'react-test-renderer';
import Radio from '../../components/Radio';

it('renders correctly', () => {
  const tree = renderer.create(<Radio selected />).toJSON();
  expect(tree).toMatchSnapshot();

  const tree1 = renderer.create(<Radio selected={false} />).toJSON();
  expect(tree1).toMatchSnapshot();
});
