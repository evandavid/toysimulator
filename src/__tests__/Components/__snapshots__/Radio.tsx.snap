// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  className="sc-pGacB gBGhfQ"
>
  <View
    className="sc-jrAFXE ktuycC"
  />
</View>
`;

exports[`renders correctly 2`] = `
<View
  className="sc-pGacB gBGhfQ"
/>
`;
